docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -260,6 +328,8 @@ Standard TensorFlow\* operations:
 | TopkV2 | No |
 | Transpose | No |
 | Unpack | No |
+| Variable | No |
+| VariableV2 | No |
 | Where | No |
```

Comment:
Where - "Supported only when it is part of a sub-graph of the special form" because in general case we have to support dynamism

Reply:
Done.